sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -84,7 +84,7 @@ public int MaxConcurrentSessions
             }
         }
 
-        /// <summary>Gets or sets whether the autocomplete option of the session handler is enabled.</summary>
+        /// <summary>Gets or sets whether the autocomplete option for messages in the session handler is enabled.</summary>
         /// <value>true if the autocomplete option of the session handler is enabled; otherwise, false.</value>
         public bool AutoComplete { get; set; }
```

Comment:
>AutoComplete [](start = 20, length = 12)

Could you also describe the default values in each of the property?

Reply:
The default values for each property is actually documented in the above few lines of the document. I am also adding them in each <value> tag.